css-view-transitions-1/Overview.bs

```diff
@@ -747,7 +743,7 @@ The {{ViewTransition/domUpdated}} [=getter steps=] are to return [=this's=] [=Vi
 
             Note: This happens if |transition| was [=skip the page transition|skipped=] before this point.
 
-        1. [=Call the DOM update callback=] of |transition|.
+        1. If it is not null, [=Call the DOM update callback=] of |transition|. Otherwise, treat |transition|'s [=ViewTransition/DOM updated promise] in subsequent steps as resolved.
```

Comment:
Could you update the text here: https://drafts.csswg.org/css-view-transitions-1/#call-the-dom-update-callback to resolve the [DOM updated promise](https://drafts.csswg.org/css-view-transitions-1/#viewtransition-dom-updated-promise) if the callback is null.

Reply:
Done.